src/index.html

```diff
@@ -11,12 +11,105 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="./reset.css"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        class="header_item logo"
+        src="images/logo.png"
+        alt="logo"
+      />
+
+      <nav class="header_item nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+              data-qa="hover"
+            >
+              Laptops &amp; Computers
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="index.html"
```

Comment:
All navigation links currently point to 'index.html'. If this is not intentional, you should update the 'href' attributes to point to the correct pages.